### ISSUE TITLE
Component doesn't updating after changing svg file.

### ISSUE THREAD

**Issue by MrFoxPro** [jfgodoy/vite-plugin-solid-svg#16]
I need to manually restart vite (v3.0. 4) to apply changes.
My config: 
```ts
 solidSvg({
    defaultExport: 'component',
 }),
 solid({
    hot: dev,
    dev: dev,
    ssr: false,
 }),
```

**Comment by jfgodoy** [comment]
I'm investigating this issue. In one of my examples it is updated immediately, and in another by refreshing the page. But in no case I had to restart the vite.

**Comment by jfgodoy** [comment]
hi @MrFoxPro, you were right, the hot module reload wasn't working. I had a hard work figuring out how to fix it, but I finally found a solution that even simplifies the code.

The fix was published as 0.4.1

**Comment by MrFoxPro** [comment]
Thanks
